dashboard/app/assets/javascripts/callouts.js

```diff
@@ -25,7 +25,16 @@
    * @param {CalloutDefinition[]} callouts
    */
   dashboard.createCallouts = function (callouts) {
-    if (!callouts) {
+    callouts = callouts.filter(function (element, index, array) {
```

Comment:
I believe you still need to protect against callouts = null, otherwise callouts.filter will blow up.